pkg/virt-controller/services/template.go

```diff
@@ -62,6 +70,79 @@ func (t *templateService) RenderLaunchManifest(vm *v1.VM) (*kubev1.Pod, error) {
 	return &pod, nil
 }
 
+func (t *templateService) RenderMigrationJob(vm *v1.VM, sourceNode *kubev1.Node, targetNode *kubev1.Node) (*batch.Job, error) {
+	srcAddr := ""
+	dstAddr := ""
+	for _, addr := range sourceNode.Status.Addresses {
```

Comment:
Ah! I missed that there was no "break" in the second case. Given that, I strongly prefer your approach as it only makes a single pass through the list.

clever.